tdtax/phenomenological.yaml

```diff
@@ -0,0 +1,34 @@
+class: Phenomenological
+tags: [Time series characteristics]
+subclasses:
+  - class: non-variable
+  - class: variable
+  - class: periodic
+    subclasses:
+      - class: sinusoidal
+        subclasses:
+          - class: ellipsoidal
+      - class: sawtooth
+  - class: multi periodic
+  - class: long timescale
+  - class: irregular
+  - class: eclipsing
+    subclasses:
+      - class: EA
+      - class: EB
+      - class: EW
+  - class: flaring
+  - class: dipping
+  - class: bogus
+    subclasses:
+      - class: galaxy
+      - class: blend
+      - class: bright star
+      - class: ccd artifact
+  - class: wrong period
+    subclasses:
+      - class: half period
+      - class: double period
+  - class: nice
+    subclasses:
+      - class: niice
```

Comment:
What does this mean?

Reply:
These labels were used by the human classifiers of the golden dataset to identify light curves that represented the archetype of their classification. 'nice' and 'niice' are only used 15 and 4 times respectively in the golden dataset, so it may be appropriate to exclude these labels from the taxonomy (and upload them as comments to Fritz instead).

Reply:
Yes, please remove nice/niice from the taxonomy. I'd rather have the taxonomy pristine and the data it is applied to cleaned up appropriately.

Reply:
Agreed - I just pushed a new version without these two labels.